pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -197,23 +197,23 @@ func (pl *CSILimits) getCSIDriverInfo(csiNode *storagev1.CSINode, pvc *v1.Persis
 
 		pluginName, err := pl.translator.GetInTreePluginNameFromSpec(pv, nil)
 		if err != nil {
-			klog.V(5).Infof("Unable to look up plugin name from PV spec: %v", err)
+			klog.V(5).ErrorS(err, "Unable to look up plugin name from PV spec")
```

Comment:
Add the PV name.

Reply:
Keep them as original log level, which is info.

Reply:
Thanks, done.